src/PlacesAutocomplete.js

```diff
@@ -49,7 +49,13 @@ class PlacesAutocomplete extends React.Component {
   }
 
   autocompleteCallback(predictions, status) {
-    if (status != this.autocompleteOK) { console.error('place autocomplete failed'); return; }
+    if (status != this.autocompleteOK) {
+      this.props.onError()
+      if (this.props.clearItemsOnError) {
+        this.setState({ autocompleteItems: [] })
```

Comment:
Thanks! I totally forgot about `this.clearAutocomplete()`, even though it's right below this code 😆 

Re: passing status to the user, that's a great idea. It looks like there are seven status codes other than `OK` that can be returned, so it would definitely be useful to get some more detail about the error and allow for different handling based on the status code.